modules/fts/pages/fts-sorting.adoc

```diff
@@ -77,6 +77,8 @@ This query sorts search-results based on `reviews.ratings.Overall` — a field t
 When there are multiple values, the highest `Overall` ratings are used for sorting.
 Hotels with no `Overall` rating are placed at the end.
 
+It is highly recommended to specify the `type` of the sort fields like `string` for text fields, `date` for datetime fields or `number` for numeric fields while mentioning the sort fields in the sort section of the search request. This would help in fetching more accurate sort results. 
```

Comment:
Suggested reword: 
To fetch more accurate sort results, we strongly recommended specifying the `type` of the sort fields in sort section of the search result. For example, `string` for text fields, `date` for datetime fields, or `number` for numeric fields.

Reply:
@sreekanth-cb Please let me know if the suggested reword looks ok.

Reply:
two minor fixes i guess..
"we strongly **recommend**" and "in sort section of the search **request**" ?

Reply:
Thanks @amarantha-k , Fixed it and added a few new lines to improve readability.. Let me know if that is an issue.